dronegen/tests.go

```diff
@@ -54,8 +54,7 @@ func testCodePipeline() pipeline {
 				`mkdir -p /tmpfs/go/src/github.com/gravitational/teleport /tmpfs/go/cache`,
 				`cd /tmpfs/go/src/github.com/gravitational/teleport`,
 				`git init && git remote add origin ${DRONE_REMOTE_URL}`,
-				`
-# handle pull requests
+				`# handle pull requests
```

Comment:
These changes (removing the initial newline) are just to make the formatting in `.drone.yml` a bit nicer as VS Code's YAML syntax highlighting seems to break when there's a newline in a YAML multiline `- |2` block.

It basically changes this:

```yaml
- |2

  git checkout...
```

to this:
```yaml
- |
  git checkout
```